From 19cab11b3a07515a3ad84309ee540c4a207bbc66 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 16:43:55 +0200
Subject: [PATCH 38/73] prefs: block third-party cookies by default

Reject extra cookies which are likely used for tracking and whatnot.
There are very few websites which require this, and if you do visit
some, static exceptions can be defined.
---
 chrome/browser/prefs/browser_prefs.cc | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git chrome/browser/prefs/browser_prefs.cc chrome/browser/prefs/browser_prefs.cc
index 2d72455654132..6cd41f4d69c6b 100644
--- chrome/browser/prefs/browser_prefs.cc
+++ chrome/browser/prefs/browser_prefs.cc
@@ -600,9 +600,7 @@ void RegisterProfilePrefsForMigration(
   registry->RegisterDictionaryPref(kHashedAvailablePages);
 
   registry->RegisterDictionaryPref(kObservedSessionTime);
-
-  registry->RegisterBooleanPref(kBlockThirdPartyCookies, false);
-
+  registry->RegisterBooleanPref(kBlockThirdPartyCookies, true);
   registry->RegisterTimePref(kPluginsDeprecationInfobarLastShown, base::Time());
 
   registry->RegisterIntegerPref(kPasswordManagerOnboardingState, 0);
-- 
2.32.0

